frontend/src/pages/Search.js

```diff
@@ -81,9 +83,13 @@ function Search({ location, history }) {
   const [defaultSearch, setDefaultSearch] = useState(true);
   // Search text for the default search if search bar was used
   const searchTextInput = useRef(null);
-  const [searchText, setSearchText] = useState('');
+  setSSObjectIfEmpty('searchText', '');
+  const [searchText, setSearchText] = useState(getSSObject('searchText'));
   // Is the search form  collapsed?
-  const [collapsed_form, setCollapsedForm] = useState(false);
+  setSSObjectIfEmpty('collapsed_form', false);
+  const [collapsed_form, setCollapsedForm] = useState(
+    getSSObject('collapsed_form')
+  );
```

Comment:
could make `getSSObject` take a second argument with the default value - that way we can combine `setSSObjectIfEmpty` with `getSSObject`

Reply:
good idea